filefn/examples/production/server/.env.example

```
@@ -0,0 +1,9 @@
+DATABASE_URL=postgres://postgres:postgres@localhost:5432/filefn_demo
+AWS_REGION=us-east-1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the DATABASE_URL key

(UnorderedKey)